### PR TITLE
conform, github-markdown-toc, jmespath, scenery, tflint

### PR DESCRIPTION
**Pull request by skyzyx**
| Language            | Repository                                                                            | CI Status                                                                                                                                         |
|:--------------------|:--------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------|
| Conform             | [skyzyx/asdf-conform](https://github.com/skyzyx/asdf-conform)                         | [Build Status](https://travis-ci.org/skyzyx/asdf-conform)                         |
| GitHub Markdown ToC | [skyzyx/asdf-github-markdown-toc](https://github.com/skyzyx/asdf-github-markdown-toc) | [Build Status](https://travis-ci.org/skyzyx/asdf-github-markdown-toc) |
| JMESPath            | [skyzyx/asdf-jmespath](https://github.com/skyzyx/asdf-jmespath)                       | [Build Status](https://travis-ci.org/skyzyx/asdf-jmespath)                       |
| Scenery             | [skyzyx/asdf-scenery](https://github.com/skyzyx/asdf-scenery)                         | [Build Status](https://travis-ci.org/skyzyx/asdf-scenery)                         |
| TFLint              | [skyzyx/asdf-tflint](https://github.com/skyzyx/asdf-tflint)                           | [Build Status](https://travis-ci.org/skyzyx/asdf-tflint)                           |

### CODE REVIEW

**Comment by skyzyx** [comment]
Looks like there are errors with other plugins, but none of these are on that list.

File: plugins/conform [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/skyzyx/asdf-conform
```
Comment (imbsky) [inline plugins/conform]: This should be the URL to the Git repository itself, which for GitHub project would be `https://github.com/skyzyx/asdf-conform.git`.

File: plugins/github-markdown-toc [hunk 2]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/skyzyx/asdf-github-markdown-toc
```
Comment (imbsky) [inline plugins/github-markdown-toc]: Here is the same problem as above.

File: plugins/jmespath [hunk 3]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/skyzyx/asdf-jmespath
```
Comment (imbsky) [inline plugins/jmespath]: Same.

File: plugins/scenery [hunk 4]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/skyzyx/asdf-scenery
```
Comment (imbsky) [inline plugins/scenery]: Same.

File: plugins/tflint [hunk 5]
```diff
@@ -1 +1 @@
-repository = https://github.com/RykHawthorn/asdf-tflint.git
+repository = https://github.com/skyzyx/asdf-tflint
```
Comment (imbsky) [inline plugins/tflint]: Same.

Comment (imbsky) [reply plugins/tflint]: FYI @RykHawthorn

**Comment by skyzyx** [comment]
Done.

**Comment by imbsky** [comment]
Thank you! and sorry for the late response! I'm a little busy this week and next week. But I will try to review those this weekend.

**Comment by skyzyx** [comment]
No worries. I appreciate the time and energy that you volunteer to this project. The world's not going to end over some `asdf` plugins, so don't stress.

Thanks again!

**Comment by imbsky** [comment]
Thank you for your kind words! The code looks good to me! Could you resolve the README conflict?

**Comment by skyzyx** [comment]
Done.

**Comment by imbsky** [comment]
Thanks @skyzyx for your contribution!

**Pull request merged**